met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -49,7 +49,7 @@ export const PreviewBanner = () => {
                         <Grid item container direction="row" rowSpacing={1}>
                             <MetBody>
                                 This engagement is scheduled to go live on
-                                {' ' + scheduledDate + ' at ' + scheduledTime}.{' '}
+                                {' ' + scheduledDate + ' at ' + scheduledTime + ' PST'}.{' '}
```

Comment:
scheduledDate + ' at ' + scheduledTime + ' PST' 
its kind of repeating at two places..is it easy to make it a method?